Flake8 tests fail on master

Flake8 tox tests fail on master as of now.

## Logs
```
TOXENV=flake8 tox
flake8 create: /private/tmp/typeguard/.tox/flake8
flake8 installdeps: flake8
flake8 installed: flake8==3.8.1,importlib-metadata==1.6.0,mccabe==0.6.1,pycodestyle==2.6.0,pyflakes==2.2.0,zipp==3.1.0
flake8 run-test-pre: PYTHONHASHSEED='1419916053'
flake8 run-test: commands[0] | flake8 typeguard.py tests
typeguard.py:0:1: E902 FileNotFoundError: [Errno 2] No such file or directory: 'typeguard.py'
ERROR: InvocationError for command /private/tmp/typeguard/.tox/flake8/bin/flake8 typeguard.py tests (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   flake8: commands failed
```

## Reproduction:
```bash
$ git clone https://github.com/agronholm/typeguard.git
$ cd typeguard
$ TOXENV=flake8 tox
```

## Cause
Master was last tested using `flake8==3.7.9`, however tox is automatically installinig `3.8.1`.

As it turns out, `flake8 typeguard.py` does nothing and this was a bug in `flake8==3.7.9`. You can invoke `flake8` with any arguments that are not Python files or packages and it will exit with success. The proper command, `flake8 typeguard` in fact shows that linting errors have made it into master:

```
$ flake8 typeguard
typeguard/__init__.py:32:5: F401 'typing._Protocol as Protocol' imported but unused
```

--- a/tox.ini
+++ b/tox.ini
@@ -10,5 +10,5 @@ commands = python -m pytest {posargs}
 
 [testenv:flake8]
 deps = flake8
-commands = flake8 typeguard.py tests
+commands = flake8 typeguard tests
 skip_install = true

--- a/typeguard/__init__.py
+++ b/typeguard/__init__.py
@@ -27,11 +27,6 @@ except ImportError:
     Literal = TypedDict = None
 
 try:
-    from typing import Protocol
-except ImportError:
-    from typing import _Protocol as Protocol
-
-try:
     from typing import AsyncGenerator
 except ImportError:
     AsyncGenerator = None

